linkchecker

```diff
@@ -331,7 +331,7 @@ group.add_argument("--ignore-url", action="append", metavar="REGEX",
 """Only check syntax of URLs matching the given regular expression.
  This option can be given multiple times."""))
 group.add_argument("--ignore-class", action="append",
-                 dest="ignoreclass", help=_(
+                 dest="ignoreclasses", help=_(
 """Ignore any anchor links styled with this class name"""))
```

Comment:
The help text should mention that multiple class names can be specified, separated by commas